### PR TITLE
Feature/14956 Adapt privacy notice for Release 3.2

### PR DESCRIPTION
**Pull request by flxschmidt**
## Description
Adapt privacy notice for Release 3.2:

- DE
- EN
- TR
- PL (english version)
- RO (english version)
- BGR (english version)

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14956

### CODE REVIEW

**Review by 30mar — APPROVED** [review]

**Review by OLVI5 — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html [hunk 1]
```diff
@@ -9,12 +9,120 @@
...
+			per E-Mail an: <a href="mailto:datenschutz@rki.de">datenschutz@rki.de</a>.
+		</p>
+
+		<hr>
+
+		<h1>
+			Datenschutzerklärung
+		</h1>
+		<p>
+			Version 3.20 (gültig bis 30.04.2023)
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html:121]:
```suggestion
			Version 3.2.0 (gültig bis 30.04.2023)
```

File: src/xcode/ENA/ENA/Resources/Localization/bg.lproj/privacy-policy.html [hunk 2]
```diff
@@ -9,12 +9,115 @@
...
+			the RKI’s official data protection officer by post to: Robert Koch-Institut, FAO the data
+			protection officer, Nordufer 20, 13353 Berlin, or by emailing: <a
+				href="mailto:datenschutz@rki.de">datenschutz@rki.de</a>.
 		</p>
+
+		<hr>
+
+		<h1>Privacy notice</h1>
 		<p>
+			Version 3.20 (valid until 30 April 2023)
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Resources/Localization/bg.lproj/privacy-policy.html:118]:
```suggestion
			Version 3.2.0 (valid until 30 April 2023)
```

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/privacy-policy.html [hunk 3]
```diff
@@ -9,12 +9,115 @@
...
+			the RKI’s official data protection officer by post to: Robert Koch-Institut, FAO the data
+			protection officer, Nordufer 20, 13353 Berlin, or by emailing: <a
+				href="mailto:datenschutz@rki.de">datenschutz@rki.de</a>.
 		</p>
+
+		<hr>
+
+		<h1>Privacy notice</h1>
 		<p>
+			Version 3.20 (valid until 30 April 2023)
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/privacy-policy.html:118]:
```suggestion
			Version 3.2.0 (valid until 30 April 2023)

```

File: src/xcode/ENA/ENA/Resources/Localization/pl.lproj/privacy-policy.html [hunk 4]
```diff
@@ -9,12 +9,115 @@
...
+			the RKI’s official data protection officer by post to: Robert Koch-Institut, FAO the data
+			protection officer, Nordufer 20, 13353 Berlin, or by emailing: <a
+				href="mailto:datenschutz@rki.de">datenschutz@rki.de</a>.
 		</p>
+
+		<hr>
+
+		<h1>Privacy notice</h1>
 		<p>
+			Version 3.20 (valid until 30 April 2023)
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Resources/Localization/pl.lproj/privacy-policy.html:118]:
```suggestion
			Version 3.2.0 (valid until 30 April 2023)
```

File: src/xcode/ENA/ENA/Resources/Localization/ro.lproj/privacy-policy.html [hunk 5]
```diff
@@ -9,12 +9,115 @@
...
+			the RKI’s official data protection officer by post to: Robert Koch-Institut, FAO the data
+			protection officer, Nordufer 20, 13353 Berlin, or by emailing: <a
+				href="mailto:datenschutz@rki.de">datenschutz@rki.de</a>.
 		</p>
+
+		<hr>
+
+		<h1>Privacy notice</h1>
 		<p>
+			Version 3.20 (valid until 30 April 2023)
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Resources/Localization/ro.lproj/privacy-policy.html:118]:
```suggestion
			Version 3.2.0 (valid until 30 April 2023)
```

File: src/xcode/ENA/ENA/Resources/Localization/tr.lproj/privacy-policy.html [hunk 6]
```diff
@@ -9,13 +9,115 @@
...
+			görevlisi), Nordufer 20, 13353 Berlin veya e-posta yoluyla: <a href="mailto:datenschutz@rki.de">datenschutz@rki.de</a>.
+		</p>
+
+		<hr>
+
+		<h1>
+			Veri Gizliliği Beyanı
+		</h1>
+		<p>
+			Sürüm 3.20 (30.04.2023 tarihine kadar geçerlidir)
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Resources/Localization/tr.lproj/privacy-policy.html:116]:
```suggestion
			Sürüm 3.2.0 (30.04.2023 tarihine kadar geçerlidir)
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Attention! The version number is wrong!

**Comment by Ein-Tim** [comment]
@flxschmidt Did you have a look at my review?

**Pull request merged**